composer/core/engine.py

```diff
@@ -159,14 +163,36 @@ class Engine():
         state (State): The initial :class:`.State` of the trainer. ``state`` will be modified in-place.
         logger (Logger): A :class:`.Logger` instance to be used for logging algorithm and callback
             specific metrics.
+        algorithm_passes ([AlgorithmPass | Tuple[AlgorithmPass, int] | Sequence[AlgorithmPass | Tuple[AlgorithmPass, int]], optional):
+            A sequence of algorithm_passes to be registered in Engine.
```

Comment:
Could you make the documentation here more similar to what the trainer documentation says? That is, also mention here that the passes that get passed in are merged with the default passes.